cs/test/core/Core.csproj

```diff
@@ -1,116 +1,118 @@
+    <Compile Include="Random.cs" />
+    <Compile Include="SerializationTests.cs" />
+    <Compile Include="SerializerGeneratorFactoryTests.cs" />
+    <Compile Include="GenericsTests.cs" />
+    <Compile Include="StreamTests.cs" />
+    <Compile Include="Structs.cs" />
+    <Compile Include="TypeAliasTests.cs" />
+    <Compile Include="ProtocolTests.cs" />
+    <Compile Include="Util.cs" />
+    <Compile Include="XmlTests.cs" />
+    <BondCodegen Include="Aliases.bond">
+      <Options Condition="'$(TargetFrameworkVersion)' == 'v4.5'">$(BondOptions) --using="Lazy=Lazy&lt;{0}&gt;" --using="OrderedSet=SortedSet&lt;{0}&gt;" --using="Decimal=decimal" --using="EnumString=Alias.EnumString&lt;{0}&gt;" --using="Array={0}[]" --using=ArrayBlob=byte[] --using="SynchronizedList=SynchronizedCollection&lt;{0}&gt;"</Options>
+      <Options Condition="'$(TargetFrameworkVersion)' == 'v4.0'">$(BondOptions) --using="Lazy=Lazy&lt;{0}&gt;" --using="OrderedSet=SortedSet&lt;{0}&gt;" --using="Decimal=decimal" --using="EnumString=Alias.EnumString&lt;{0}&gt;" --using="Array={0}[]" --using=ArrayBlob=byte[]</Options>
+    </BondCodegen>
+    <BondCodegen Include="NamespaceConflict.bond" />
+    <BondCodegen Include="NamespaceConflictBond.bond" />
+    <BondCodegen Include="ReadOnly.bond">
+      <Options Condition=" '$(Configuration)' != 'Fields' ">$(BondOptions) --readonly-properties</Options>
+      <Options Condition=" '$(Configuration)' == 'Fields' ">--readonly-properties --collection-interfaces</Options>
+    </BondCodegen>
+    <BondCodegen Include="Containers.bond">
+      <Options Condition=" '$(Configuration)' != 'Fields' ">$(BondOptions) --collection-interfaces</Options>
+      <Options Condition=" '$(Configuration)' == 'Fields' ">--fields</Options>
+    </BondCodegen>
+    <BondCodegen Include="UnitTest.bond">
+      <Options>$(BondOptions) --using="DateTime=System.DateTime"</Options>
+    </BondCodegen>
+    <BondCodegen Include="import dir with spaces\Bond File With Spaces.bond" />
+    <!-- Resharper Workaround -->
+    <Compile Include="$(IntermediateOutputPath)\UnitTest_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\Containers_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\Aliases_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\ReadOnly_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\NamespaceConflict_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\NamespaceConflictBond_types.cs" Condition="False" />
+    <Compile Include="$(IntermediateOutputPath)\Bond File With Spaces_types.cs" Condition="False" />
+    <!-- End Resharper Workaround -->
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />

```

Comment:
This addition is unexpected. Was it intentional? Can we change the tests to just use NUnit?